### DISCUSSION TITLE
Articles are not fetched completely

### DISCUSSION THREAD

**Discussion by H3T-Shah** [spacecowboy/Feeder#330] [Polls]
Has anyone experienced same problem.

I was reading this [article](https://screenrant.com/demon-slayer-saddest-deaths-manga/) but in the feeder article was showed uptill here:
![Screenshot_2024-07-05-21-42-42-07_56dde218d5b132d57db5f243d6cd7745](https://github.com/spacecowboy/Feeder/assets/135018516/7af0ce85-4584-44bf-a2c9-60ebe3e6728b)

But opening it on browser it was all fine.
Has anyone encountered the same issue?

**Comment by H3T-Shah** [comment]
This happened again for this [article.](https://screenrant.com/best-anime-for-beginners/)

**Comment by spacecowboy** [comment]
full article parsing is not a deterministic thing.

for some sites it just fails.

**Reply by H3T-Shah** [reply]
so is it the site who's limiting it?
EDIT - it happened for same site twice so maybe it's site prob

**Reply by spacecowboy** [reply]
no its the algorithm that cant identify where the article starts or ends correctly

**Reply by H3T-Shah** [reply]
Can we fix it then?

**Reply by spacecowboy** [reply]
*you* can feel free to look into it. the code is here: https://github.com/dankito/Readability4J

**Comment by travelvice** [comment]
@spacecowboy, Would that also be why so many articles parse as pretty much just an "x" for Slashdot? It's not all of them, but the majority.

https://rss.slashdot.org/Slashdot/slashdotMain

![Screenshot_20240718_220222_Feeder](https://github.com/user-attachments/assets/7a8047fe-0346-4529-aea5-60b811542ec2)

**Reply by H3T-Shah** [reply]
Potentially, Yes

**Reply by spacecowboy** [reply]
yes
